txp/Pos/Txp/Network.hs

```diff
@@ -1,10 +1,10 @@
 -- | Txp communication layer re-exports.
 
 module Pos.Txp.Network
-       ( module Pos.Txp.Network.Listeners
-       , module Pos.Txp.Network.Types
+       ( module Pos.Txp.Network.Types
+       , module Pos.Txp.Network.Listeners
        ) where
 
 import           Pos.Arbitrary.Txp.Network ()
-import           Pos.Txp.Network.Listeners
 import           Pos.Txp.Network.Types
+import           Pos.Txp.Network.Listeners
```

Comment:
Aren't you using `stylish-haskell`? 🤔

Reply:
No

Reply:
You must (though this particular thing is minor, of course): https://github.com/input-output-hk/cardano-sl-style-guides/blob/master/haskell-style-guide.md

Reply:
I don't think it's a good idea to make it mandatory to use some code formatting tool. I'd say it's recommended, but not mandatory.

Reply:
Sure, I also didn't mean that using any particular tool is obligatory, but sorting imports is. NVM.